### PR TITLE
new package: gtar 1.32

### PR DESCRIPTION
**Pull request by a-schaefers**
## Checklist

- [x] Latest upstream version.
- [x] I agree to maintain this package (**required**).
- [x] I agree to notify KISS if I can no longer maintain this package (**required**).

### CODE REVIEW

**Comment by a-schaefers** [comment]
this should be ready

**Comment by a-schaefers** [comment]
rename gnutar ==> gtar

all of our gnu utilities are now following the same conventions FreeBSD uses

File: community/gtar/build [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/sh -e
+
+export FORCE_UNSAFE_CONFIGURE=1
+export CC="gcc -static"
+
+./configure \
+    --prefix=/usr \
+    --program-prefix=g
+
+make DESTDIR="$1" install
```
Comment (dylanaraps) [inline community/gtar/build]: Missing standalone `make`?

Comment (a-schaefers) [reply community/gtar/build]: done

**Pull request merged**